services/api-server/tests/mocks/delete_file.json

```diff
@@ -0,0 +1,105 @@
+[
+  {
+    "name": "POST /simcore-s3/files/metadata:search",
```

Comment:
Question: from the file name I would expect only the call for delete here. why is the one for searching as well in there?

Reply:
This is because the Api server searches for the file to get the right identifier and then deletes it (via storage). I think the source of these "double-calls" is that the identifier we use on the api server isn't exactly the same one as the one which is used in storage. So we always/often have to search for a file before being able to do anything with it. IMO it would be more elegant if we didn't have to do this, but C’est la vie